tools/spelling/spelling_dictionary.txt

```diff
@@ -401,7 +401,9 @@ XFF
 XML
 XN
 XNOR
+XRay
 XSS
+Xray
```

Comment:
This is definitely a problem, we shouldn't be using 3+ different terms. But if we were agnostic about exchanging these so liberally, then refer to lines 1-4 of this file which describe how case of the terms affects variant capitalization. "XRAY" would permit any casing and would be preferred to having multiple entries. Using one proper term consistently is preferred of that.

Reply:
hi @wrowe, thanks for taking a look this! my mistake for adding duplicate xray entries to the dictionary. I realize I should just fix the terms `XRay` to `X-Ray` where they were violating the spell check so `XRAY` doesn't even have to be in the dictionary anymore.

Reply:
Fantastic, I'm not back till Tuesday but if this is cleaned up, it seems like we are good to go. Someone else can also perhaps step up and merge it before then.